index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Binhussen Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="toolbar">
+        <a href="#home" class="logo">Binhussen</a>
+        <i class="material-symbols-outlined"> menu </i>
+      </nav>
+    </header>
+    <main>
+      <section class="headline">
+        <h1 class="headline-primary">Hey there. I’m Mohammed Hussen</h1>
+        <h2 class="headline-supporting">I’m a software developer</h2>
+        <p class="headline-description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] it would be better if you add some padding to your title and paragraph in order to match the Figma design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/176547154-a27426f2-3872-4bac-a85f-99e37b7dd98e.png)
- Figma
![image](https://user-images.githubusercontent.com/61790587/176547165-46be641e-4a24-4bca-a05e-72d721254e74.png)

Reply:
Thank you @sumairq for your feedback, I will Modify It.